### PR TITLE
Add dynamic columns for daughter prongs

### PR DESCRIPTION
**Pull request by ddobrigk**

### CODE REVIEW

**Comment by ddobrigk** [comment]
@a-mathis @jgrosseo here are the dynamic columns for the prong information. Please check for any obvious typos :-D Thanks!

**Comment by jgrosseo** [comment]
It does not seem to compile in the fullCI...

File: Analysis/DataModel/include/AnalysisDataModel/StrangenessTables.h [hunk 1]
```diff
@@ -104,6 +104,19 @@ DECLARE_SOA_DYNAMIC_COLUMN(Eta, eta, //!
 DECLARE_SOA_DYNAMIC_COLUMN(Phi, phi, //!
                            [](float Px, float Py) -> float { return RecoDecay::Phi(Px, Py); });
 
+DECLARE_SOA_DYNAMIC_COLUMN(NegativePt, negativept, //!
```
Comment (jgrosseo) [inline Analysis/DataModel/include/AnalysisDataModel/StrangenessTables.h]: Please add a documenting line after //! (everywhere actually ;-))

**Review by jgrosseo — CHANGES_REQUESTED** [review]
See inline comments

**Comment by sartozza** [comment]
@ddobrigk @jgrosseo may I ask you the status on this PR?
Just to know in case I need to update the V0 selection task.
Thanks!

**Comment by ddobrigk** [comment]
@sartozza @jgrosseo documentation added, please check if more is needed! 
...ah wait: my documentation fails clang-format... sigh. One moment.

**Comment by ddobrigk** [comment]
Now fixed! Please let me know if further action is required :-D

**Comment by jgrosseo** [comment]
I fear yes...
mac is not happy
`/System/Volumes/Data/build/ci_checks/alice_ci_0/o2/sw/SOURCES/O2/6690/0/Analysis/DataModel/include/AnalysisDataModel/StrangenessTables.h:158:52: error: no member named 'NegPx' in namespace 'o2::aod::v0data'
                       v0data::NegativeEta<v0data::NegPx, v0data::NegPy, v0data::NegPz>,
                                           ~~~~~~~~^
`

**Comment by ddobrigk** [comment]
@jgrosseo, we'll get there :-D Now my mac, at least, is happy! 

Sorry for missing this mishap (I guess doing code in a beach city is by definition more error-prone, so I should probably avoid it :-D)

**Comment by ddobrigk** [comment]
@jgrosseo the new error seems to be something I am innocent of... Can you please take a look? Thanks!

**Review by jgrosseo — APPROVED** [review]

**Pull request merged**